atf_eregs/templates/regulations/generic_universal.html

```diff
@@ -0,0 +1,38 @@
+{% extends "regulations/generic_universal.html" %}
+
+{% block title %} ATF eRegulations {% endblock %}
+
+
+{% block hero %}
+<div class="hero">
+    <div class="inner-wrap group">
+      <div class="primary">
+        <h2 class="hero-header">ATF Title 27 regulations</h2>
```

Comment:
Another small one -- should this be capitalized "Regulations" or lower-cased "title"? It looks a little odd.

Reply:
I thought “Title 27” (like all the titles) was a proper noun. (The subheading is in sentence case.)

Reply:
I think you're right about "Title", it just looks a little off. If it's just me, please ignore!

Reply:
This is one of those situations where it's correct but looks wrong. Changing to title case would look less wrong but then be inconsistent. Not sure what the right call is, but I'll leave as is for now.